prowler/providers/gcp/gcp_provider.py

```diff
@@ -15,9 +15,7 @@ def __init__(
         input_project_ids: list,
     ):
         logger.info("Instantiating GCP Provider ...")
-        self.credentials, self.default_project_id = self.__set_credentials__(
-            credentials_file
-        )
+        self.credentials, _ = self.__set_credentials__(credentials_file)
```

Comment:
I think it'll be better to catch the return from the `__set_credentials__` within that function and just return the `credentials` object.